### DISCUSSION TITLE
Problem with PiP & Crop

### DISCUSSION THREAD

**Discussion by kaioyoca** [WindowTop/WindowTop-App#391] [General]
Hi, I finally found an app that does what I needed, that is, crop a window as you want and interact with it without problems.
However, with the Crop mode when I crop a window I can continue to click on the cut window, but the mouse is moved about 1.5 cm upwards. This means that to click where I should click, I have to do it 1.5 cm higher as if the mouse crosshair was off-center or not calibrated. I tried on 3 different computers and I have the same error/bug. Tried on both Windows 10 and Windows 11. Please tell me how I can fix it or if it is a bug if it is possible to fix it, I can't find anything that works as well as this app. Help!

**Comment by gileli121** [comment]
@kaioyoca 

Are you experiencing this issue only when cropping is enabled? Does it work well in PiP mode with mouse interaction?

Which app are you using?

This type of bug often happens due to DPI scaling on the monitor, especially if you have multiple monitors with different DPI scaling for each one.

When I say "DPI scaling," I'm referring to these settings:
![image](https://github.com/user-attachments/assets/e0d5a028-4e19-43b9-85f5-2556b532b365)

Please let me know how many monitors you have, the DPI scaling for each, and if changing to 100% for all monitors resolves the problem. This can help determine if the issue is related to scaling.

Thanks

**Reply by kaioyoca** [reply]
Hi and thanks for the reply. So I use multiple monitors and I tried disconnecting the others to use only one but it didn't solve the problem. The resizing is 100% on all monitors, on 1 PC I use 3 and on the other 2 monitors. The problem is only when I crop and with or without mouse mode. For the apps I use in general there are so many to list, is there anyone in particular that could cause this problem? What could it be since I'm the only one it happens to? I tried on all my PCs and it gives the same error.

**Reply by kaioyoca** [reply]
I may have to format one of my PCs and maybe if I do I will try after the formatting if it solves the problem. It must be said that one of my PCs where I tried the app was formatted only a few weeks ago and has very few apps and programs installed

**Reply by kaioyoca** [reply]
I tried both on rayzen and intel pc with amd 6750xt and intel integrated graphics cards

**Comment by kaioyoca** [comment]
Hi and thanks for the reply. So I use multiple monitors and I tried disconnecting the others to use only one but it didn't solve the problem. The resizing is 100% on all monitors, on 1 PC I use 3 and on the other 2 monitors. The problem is only when I crop and with or without mouse mode. For the apps I use in general there are so many to list, is there anyone in particular that could cause this problem? What could it be since I'm the only one it happens to? I tried on all my PCs and it gives the same error.

I may have to format one of my PCs and maybe if I do I will try after the formatting if it solves the problem. It must be said that one of my PCs where I tried the app was formatted only a few weeks ago and has very few apps and programs installed

I tried both on rayzen and intel pc with amd 6750xt and intel integrated graphics cards

**Comment by kaioyoca** [comment]
Update:
unfortunately I noticed that this problem only happens with the two apps that I use with windowtop which are "WING Edit" (the software to control the digital mixer Wing Behringer) and with the app "Mixing Station" which is another app that is used to control various digital mixers.
In these apps there are various ways to resize the interface, I tried them all but it does not solve the problem.
A bad luck that happens exactly on the apps that I needed.
If you want you can download the apps and use them offline even if you do not have a mixer to test, they are free.
I am really unlucky...

**Reply by gileli121** [reply]
Thanks for the information.  
I opened an issue here: https://github.com/WindowTop/WindowTop-App/issues/393 and linked it to this discussion.  
When I have time to address this bug, I will work on it.  

Currently, I am working on tickets from #389.

**Comment by kaioyoca** [comment]
Thanks, I still purchased the license even though I only need this app for those two programs, but I'm confident that the problems will be fixed and I'm happy to support your work anyway

**Reply by gileli121** [reply]
Thanks a lot!  
Indeed, I’m continually improving the product by adding more features and fixing bugs.  
I even created a roadmap thread to document the progress: https://github.com/WindowTop/WindowTop-App/issues/389
